packages/preview/genealotree/0.1.0/examples/example.typ

```diff
@@ -0,0 +1,37 @@
+#import "../genealotree.typ": *
```

Comment:
To make this example easier to copy/paste and use as a starting point, using the package specification would be nice.
```suggestion
#import "@preview/genealotree:0.1.0": *
```